nimbus-jose-jwt_aws-kms-extension/src/main/java/com/nimbusds/jose/aws/kms/crypto/impl/KmsDefaultEncryptionCryptoProvider.java

```diff
@@ -62,13 +62,24 @@ public abstract class KmsDefaultEncryptionCryptoProvider extends PublicBaseJWEPr
     /**
      * The supported JWE algorithms (alg) by the AWS crypto provider class.
      * <p>
-     * Note: We are using KMS prescribed algorithm names here.
-     * Ref: https://docs.aws.amazon.com/kms/latest/developerguide/asymmetric-key-specs.html
+     * Note: We accept both the algorithms defined in RFC-7518 and KMS-defined strings.
+     * @see <a href="https://datatracker.ietf.org/doc/html/rfc7518#section-4.1">RFC-7518 Section 4.1</a>
+     * @see <a href="https://docs.aws.amazon.com/kms/latest/developerguide/asymmetric-key-specs.html"> KMS Asymmetric key specs </a>
      */
     public static final Set<JWEAlgorithm> SUPPORTED_ALGORITHMS = ImmutableSet.of(
             JWEAlgorithm.parse(EncryptionAlgorithmSpec.SYMMETRIC_DEFAULT.name()),
             JWEAlgorithm.parse(EncryptionAlgorithmSpec.RSAES_OAEP_SHA_1.name()),
-            JWEAlgorithm.parse(EncryptionAlgorithmSpec.RSAES_OAEP_SHA_256.name()));
+            JWEAlgorithm.parse(EncryptionAlgorithmSpec.RSAES_OAEP_SHA_256.name()),
+            JWEAlgorithm.RSA_OAEP_256,
+            JWEAlgorithm.RSA_OAEP);
```

Comment:
It seems there more algorithms defined in [JWEAlgorithm](https://www.javadoc.io/static/com.nimbusds/nimbus-jose-jwt/9.31/com/nimbusds/jose/JWEAlgorithm.html), don't we need to support other algorithms?